## This is the blob information
## Related pkg: visor
##
## Author: Ferdian Pratama

Header header
int16 count
#######int8[] parent
#######int8[] color
#######int32[] area
#######int32[] perimeter
#######float32[] sumx
#######float32[] sumy
int16[] minx
int16[] maxx
int16[] miny
int16[] maxy
#sensor_msgs/Image scene
